Client/Frontend/Browser/TopTabCell.swift

```diff
@@ -109,8 +94,30 @@ class TopTabCell: UICollectionViewCell, NotificationThemeable, TabTrayCell, Reus
         layer.zPosition = CGFloat(layoutAttributes.zIndex)
     }
 
-    func applyTheme() {
-        selectedBackground.backgroundColor = UIColor.theme.topTabs.tabBackgroundSelected
+    func applySelectedStyle(theme: Theme) {
+        titleText.textColor = theme.colors.textPrimary
+        closeButton.tintColor = theme.colors.textPrimary
+
+        selectedBackground.backgroundColor = theme.colors.layer2
+        selectedBackground.layer.shadowColor = theme.colors.shadowDefault.cgColor
+        selectedBackground.isHidden = false
+    }
+
+    func applyUnselectedStyle(theme: Theme) {
+        titleText.textColor = theme.colors.textPrimary
+        closeButton.tintColor = theme.colors.textPrimary
+
```

Comment:
Added two different methods for selected/unselected states even tho we use the same colors. In the future, it will be easier to distinguish between the two states.